src/client/Microsoft.Identity.Client/PlatformsCommon/Interfaces/IKeyMaterialManager.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Identity.Client.PlatformsCommon.Shared;
+
+namespace Microsoft.Identity.Client.PlatformsCommon.Interfaces
+{
+    internal interface IKeyMaterialManager
```

Comment:
Since this interface is strongly tied to MSI, I would rename it, maybe `SlcKeyAndCertManager` or smth.